docs/reference/edgeql/index.rst

```diff
@@ -81,6 +85,7 @@ Introspection command:
 
     select
     for
```

Comment:
For consistency with the non-reference docs let's move `for` and `group` underneath `delete`?